project-base/storefront/components/Layout/Header/MenuIconic/MenuIconicElements.tsx

```diff
@@ -0,0 +1,67 @@
+import { Icon } from 'components/Basic/Icon/Icon';
+import { IconName } from 'components/Basic/Icon/IconsSvgMap';
+import { twMergeCustom } from 'utils/twMerge';
+import { forwardRef } from 'react';
+import { ExtendedNextLink } from 'components/Basic/ExtendedNextLink/ExtendedNextLink';
+
+export const MenuIconicItemIcon: FC<{ icon: IconName }> = ({ icon, className }) => (
+    <Icon iconType="icon" icon={icon} className={twMergeCustom('mr-2 w-4 text-white', className)} />
+);
+
+export const MenuIconicItem: FC = ({ children, className, dataTestId }) => (
+    <li className={twMergeCustom('relative mr-5 flex last:mr-0 xl:mr-8', className)} data-testid={dataTestId}>
+        {children}
+    </li>
+);
+
+type MenuIconicItemLinkProps = { onClick?: () => void; href?: string };
+
+export const MenuIconicSubItemLink: FC<MenuIconicItemLinkProps> = ({ children, href, onClick, dataTestId }) => {
+    const content = (
+        <a className="block py-3 px-5 text-sm text-dark no-underline" data-testid={dataTestId} onClick={onClick}>
+            {children}
+        </a>
+    );
+
+    if (href) {
+        return (
+            <ExtendedNextLink href={href} passHref type="static">
+                {content}
+            </ExtendedNextLink>
+        );
+    }
+
+    return content;
+};
+
+export const MenuIconicItemLink: FC<MenuIconicItemLinkProps> = forwardRef(
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    ({ children, className, dataTestId, href, onClick }, _) => {
```

Comment:
I have seen this in multiple places, but can't you just do this?

```suggestion
    ({ children, className, dataTestId, href, onClick }) => {
```

Reply:
Unfortunately no, because React is still complaining. 

![image](https://github.com/shopsys/shopsys/assets/11465991/58a281e5-e35f-4151-931c-bea43cbff89c)

Reply:
OMG :upside_down_face: